src/features/prefixes.ts

```diff
@@ -0,0 +1,6 @@
+export enum PrefixesEnum {
```

Comment:
prefixes but what for?
there could be a prefixes for almost anything so there is a lack of context here.
A prefix for what?

Reply:
Will do!